system/BaseModel.php

```diff
@@ -1344,7 +1344,9 @@ public function validate($data): bool
             return true;
         }
 
-        return $this->validation->setRules($rules, $this->validationMessages)->run($data, null, $this->DBGroup);
+        $this->validation->reset()->setRules($rules, $this->validationMessages);
+
+        return $this->validation->run($data, null, $this->DBGroup);
```

Comment:
Aside from adding the `->reset()` method, may I know why the line got broken into two? Or was this because of line length?

Reply:
Yes, it is too long.
And one line is a bit difficult to debug, step debugging.